test/CMakeLists.txt

```diff
@@ -1,2 +1,5 @@
-add_test(NAME tuple_keydef.test.lua
-         COMMAND runtest.sh tuple_keydef.test.lua)
+add_test(
+    NAME tuple_keydef.test.lua
+    COMMAND runtest.sh tuple_keydef.test.lua
+    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
If you don't mind, I'll move it into its own commit. [It seems](https://github.com/tarantool/tuple-keydef/pull/10#discussion_r639017243), the change fixes test running, when out-of-source build is used, and, strictly speaking, is neither about CI, nor about hot reload.

Reply:
Moved to PR #12.